src/debugger/Debugger.js

```diff
@@ -116,29 +127,37 @@ export class DebugServer {
       return;
     }
     let parts = command.split(" ");
-    if (parts[0] === "breakpoint") {
+    if (parts[0] === DebugMessage.BREAKPOINT) {
```

Comment:
I do not think you need this. This function should be renamed to "processDebuggerCommand()" which process any debugger command form UI. It should be a big switch case which process any individual command instead of special case for breakpoint.

Reply:
The github code collapsing confused me. You already call this executeCommand() which is good. But my later comment still applies: process all debugger UI commands as a big flat switch case, do not make hierachy commands here.